### PR TITLE
Rework config defaults.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
show help text for config fields as tooltip

### CODE REVIEW

File: openslides/agenda/config_variables.py [hunk 1]
```diff
@@ -55,30 +55,27 @@ def get_config_variables():
...
 
     yield ConfigVariable(
-        name="agenda_hide_internal_items_on_projector",
-        default_value=True,
-        input_type="boolean",
-        label="Hide internal items when projecting subitems",
-        weight=225,
+        name="agenda_item_creation",
+        label="Add to agenda",
+        default_value="default_no",
```
Comment (normanjaeckel) [inline openslides/agenda/config_variables.py]: default_yes

File: openslides/assignments/config_variables.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def get_config_variables():
 
     yield ConfigVariable(
         name="assignments_poll_100_percent_base",
-        default_value="YES_NO_ABSTAIN",
+        default_value="VALID",
```
Comment (normanjaeckel) [inline openslides/assignments/config_variables.py]: This assumes that one adds values to the valid field. But in ad hoc elections this is not always true. So I prefer YES_NO_ABSTAIN.

File: openslides/motions/config_variables.py [hunk 3]
```diff
@@ -75,7 +60,7 @@ def get_config_variables():
 
     yield ConfigVariable(
         name="motions_default_line_numbering",
-        default_value="none",
+        default_value="outside",
```
Comment (normanjaeckel) [inline openslides/motions/config_variables.py]: I don't like it, but feel free to change it.

File: openslides/agenda/config_variables.py (hunk 1 above)
Comment (emanuelschuetze) [reply openslides/agenda/config_variables.py]: Ok, changed.

File: openslides/assignments/config_variables.py (hunk 2 above)
Comment (emanuelschuetze) [reply openslides/assignments/config_variables.py]: Ok, changed.

File: openslides/motions/config_variables.py (hunk 3 above)
Comment (emanuelschuetze) [reply openslides/motions/config_variables.py]: As discussed, we will change the default to 'outside'.

**Comment by emanuelschuetze** [comment]
All remarks done. Rebased.

**Pull request merged**